examples/statdb-client/main.go

```diff
@@ -20,7 +20,7 @@ import (
 
 var (
 	port   string
-	apiKey = []byte("")
+	apiKey = ""
```

Comment:
I got the apikey using context. 
apiKey, ok := auth.GetAPIKey(ctx).   using "storj.io/storj/pkg/auth"

will this work? I recent work on irreparable Q followed statdb design. In my code review I was suggested to use as above.

Reply:
If you look in this PRs `pkg/statdb/statdb.go` file, in the `validateAuth` function, I'm using `auth.GetAPIKey` to access the api key. 
The file this comment is on is for the example statdb client, which is constructed by passing the api key in as a string. Then, inside the constructor for sdbclient, the api key is passed into the context